Hello, traveler. This is the beginning of a log file called: "Fascist Tweet Archiver".
Some basic system info about the host machine: posix.uname_result(sysname='Linux', nodename='file-server', release='3.16.0-4-686-pae', version='#1 SMP Debian 3.16.43-2+deb8u5 (2017-09-19)', machine='i686')
This log was begun Saturday, 25 November 2017 at 06:52



INFO: read target account data; we're tracking 52 accounts
WARNING: no ProtocolError (got exception "name 'ProtocolError' is not defined");
 trying requests.packages.urllib3.exceptions instead
NOTE: successfully imported from requests
Starting up...
about to get all tweets newer than ID #923261387434012672 for account
@SecretaryRoss
ERROR: unable to get tweets for account @SecretaryRoss
           the system said: Failed to send request:
HTTPSConnectionPool(host='api.twitter.com', port=443): Read timed out. (read
timeout=60)
about to get all tweets newer than ID #890659905798983680 for account @Reince45
about to get all tweets newer than ID #934195424033955840 for account
@stevenmnuchin1
ERROR: unable to get tweets for account @stevenmnuchin1
           the system said: Failed to send request: ('Connection aborted.',
BadStatusLine("''",))
about to get all tweets newer than ID #934081286351794179 for account @POTUS
ERROR: unable to get tweets for account @POTUS
           the system said: Failed to send request: ('Connection aborted.',
ConnectionResetError(104, 'Connection reset by peer'))
about to get all tweets newer than ID #-1 for account @ElaineChao
about to get all tweets newer than ID #934198554951913479 for account
@nikkihaley
ERROR: unable to get tweets for account @nikkihaley
           the system said: Failed to send request:
HTTPSConnectionPool(host='api.twitter.com', port=443): Read timed out. (read
timeout=60)
about to get all tweets newer than ID #836780744810844161 for account
@StephenMillerAL
ERROR: unable to get tweets for account @StephenMillerAL
           the system said: Failed to send request:
HTTPSConnectionPool(host='api.twitter.com', port=443): Max retries exceeded with
 url: /1.1/statuses/user_timeline.json?screen_name=StephenMillerAL&count=200
(Caused by NewConnectionError('<requests.packages.urllib3.connection.VerifiedHTT
PSConnection object at 0xb642ab2c>: Failed to establish a new connection: [Errno
 -2] Name or service not known',))
about to get all tweets newer than ID #934330330043035648 for account @Scavino45
